style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #fff;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 27px;
+  color: #acb7c3;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.slide {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px auto;
+  background-color: #acb7c3;
+}
+
+.content {
+  background-image: url(./images/mobile/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-position-x: 100%;
+}
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=page%20you%20should%20use%20flexbox%20in%20all%202%20sections.), you are required to use flexbox on this section to lay out its content. Kindly fix that